controlplane/nested/kubeadm/consts.go

```diff
@@ -0,0 +1,21 @@
+package kubeadm
+
+const (
+	KubeadmExecPath          = "/kubeadm"
+	KASManifestsPath         = "/etc/kubernetes/manifests/kube-apiserver.yaml"
```

Comment:
By using the same default file locations, could it cause conflict, for eg, when multiple clusters are created at the same time?

Reply:
Yes. That could be a problem. `kubeadm` does not support generating static pod manifests in a directory other than `/etc/kubernetes/manifests`. We should figure out a way to solve it.

Reply:
@christopherhein @Fei-Guo do you have any suggestions?

Reply:
Can we span a new pod for each cluster and generate the manifests in the new pod then terminate after they have been created as configmaps?

Reply:
> Can we span a new pod for each cluster and generate the manifests in the new pod then terminate after they have been created as configmaps?

@christopherhein @jzhoucliqr 

Yes. That's also the only way I can see. Let's try to merge this PR first and I will create a new issue/PR to fix it?

Reply:
I think that might work the only idea I could have is tracking what cluster is currently being templated and "lock" other reconciliation from processing until the current cluster bring up finishes.

Reply:
Then we need to add a big lock in the reconcile loop or reduce the ncp_controller's concurrency to one to ensure the ncp_controller is not creating more than one nested control plane at a time. Will that affect the performance?